sql/wsrep_sst.cc

```diff
@@ -82,17 +83,18 @@ static Regex sst_allowed_methods_regex;
    characters and the ways they could be used to inject the command to
    the OS. However this approach seems to be too error prone.
    Instead of this we will just allow alpha-num + a few special characters
-   (colon, slash, dot, underscore, square brackets).
+   (colon, slash, dot, underscore, square brackets, hyphen).
 
-   It is the same as ^[\w:\/\.\[\]]+$    */
+   It is the same as ^[\w:/.[\]-]+$    */
```

Comment:
Shouldn't this be
```suggestion
   It is the same as ^[\w:/.\[\]-]+$    */
```

Reply:
No, we don't need to escape opening square bracket.

Reply:
You're right! https://www.regular-expressions.info/charclass.html#special says it is not required.